standard/table.lua

```diff
@@ -67,16 +67,10 @@ end
 ---@param tbl table?
 ---@return boolean
 function Table.isEmpty(tbl)
-	if tbl == nil then
+	if not tbl then
```

Comment:
do we want to catch cases where we get a string or bool?
i.e.
`	if type(tbl) ~= 'table' then`